components/keyboard-shortcuts/test/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import KeyboardShortcuts from '../';
+
+describe( 'KeyboardShortcuts', () => {
+	afterEach( () => {
+		while ( document.body.firstChild ) {
+			document.body.removeChild( document.body.firstChild );
+		}
+	} );
+
+	function keyPress( which, target ) {
+		[ 'keydown', 'keypress', 'keyup' ].forEach( ( eventName ) => {
+			const event = new window.Event( eventName, { bubbles: true } );
+			event.keyCode = which;
+			event.which = which;
+			target.dispatchEvent( event );
+		} );
+	}
+
+	it( 'should capture key events', () => {
+		const spy = jest.fn();
+		mount(
+			<KeyboardShortcuts
+				shortcuts={ {
+					d: spy,
+				} } />
+		);
+
+		keyPress( 68, document );
```

Comment:
It took me a few seconds to connect d char with 68 :)
Can we hide 68 behind the constant?